charts/spark-hs/README.md

```diff
@@ -81,6 +81,7 @@ Spark history server UI to view the job execution details.
       --master k8s://https://<k8s-master-IP> \
       --deploy-mode cluster \
       --name spark-pi \
+      --conf spark.kubernetes.shuffle.namespace=spark \
```

Comment:
Why '...shuffle.namespace' for history server, instead of 'spark.kubernetes.namespace'?

Reply:
Copy-paste error. Should be spark.kubernetes.namespace